EIPS/eip-2400.md

```diff
@@ -0,0 +1,104 @@
+---
+eip: 2400
+title: Transaction Receipt URI
+author: Ricardo Guilherme Schmidt (@3esmit), Eric Dvorsak (@yenda)
+discussions-to: https://ethereum-magicians.org/t/eip-2400-transaction-receipt-uri/
+type: Standards Track
+category: ERC
+status: Draft
+created: 2019-11-05
+requires: 831
```

Comment:
This depends on `chain_id` (and EIP-831 doesn't). Please add EIP-155 as a dependency and refer to it in the specification section.

Reply:
Good point. Fixed it. Thanks.